editor/src/components/editor/actions/actions.tsx

```diff
@@ -902,6 +911,7 @@ export function restoreEditorState(
     refreshingDependencies: currentEditor.refreshingDependencies,
     colorSwatches: currentEditor.colorSwatches,
     internalClipboard: currentEditor.internalClipboard,
+    filesModifiedByElsewhere: currentEditor.filesModifiedByElsewhere,
```

Comment:
Ok I know I was the one that named this during the spike, but it is an awful name 😬 Can we rename it to `filesModifiedByAnotherUser` please?

Reply:
Fixed in e30d6bb6c.